packages/components/src/popover/index.js

```diff
@@ -80,24 +80,32 @@ class Popover extends Component {
 	}
 
 	componentWillUnmount() {
-		this.toggleWindowEvents( false );
-
 		clearTimeout( this.focusTimeout );
+		this.toggleAutoRefresh( false );
 	}
 
-	toggleWindowEvents( isListening ) {
-		const handler = isListening ? 'addEventListener' : 'removeEventListener';
-
-		window.cancelAnimationFrame( this.rafHandle );
-		window[ handler ]( 'resize', this.throttledComputePopoverPosition );
-		window[ handler ]( 'scroll', this.throttledComputePopoverPosition, true );
+	toggleAutoRefresh( isActive ) {
+		// Resize and scroll window events
+		const handler = isActive ? 'addEventListener' : 'removeEventListener';
+		window[ handler ]( 'resize', this.throttledRefresh );
+		window[ handler ]( 'scroll', this.throttledRefresh, true );
+
+		// Auto refresh every 0.5s
+		if ( isActive ) {
+			this.autoRefresh = setInterval( () => {
```

Comment:
Should we add a comment explaining why we're using `setInterval` (no resize and position events)

Reply:
Yup, it is not obvious. Added in https://github.com/WordPress/gutenberg/pull/11257/commits/8c5645b6d134c02227fb68bfd691c3bbb7cf0df1